cart.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cart | eshop</title>
+    <link rel="stylesheet" href="/assets/styles/style.css" />
+    <script src="/assets/scripts/custom-components.js"></script>
+  </head>
+  <body>
+    <custom-header></custom-header>
+
+    <div class="container cart-container">
+      <div class="cart-products">
+        <div class="card card-landscape">
+          <img
+            class="card-img"
+            src="https://picsum.photos/500/800"
+            alt="card image"
+          />
+
+          <div class="card-description-action">
+            <div class="card-description mb-8 mi-8">
+              <h3 class="h5">Card Heading</h3>
+              <p class="h6">$ 200</p>
+              <p class="h7">20% off</p>
+              <div class="form-group">
+                <label>
+                  <span class="label">Quantity</span>
+                  <div class="input-group">
+                    <div class="input-group-prepend">
+                      <span class="input-group-text" id="inputGroupPrepend">
+                        <i class="fas fa-plus"></i>
+                      </span>
+                    </div>
+                    <input
+                      type="number"
+                      step="1"
+                      min="1"
+                      class="form-control"
+                      placeholder="Quantity"
+                      aria-describedby="inputGroupPrepend"
+                    />
+                    <div class="input-group-append">
+                      <span class="input-group-text" id="inputGroupAppend">
+                        <i class="fas fa-minus"></i>
+                      </span>
+                    </div>
+                  </div>
+                </label>
+              </div>
+            </div>
+            <div class="card-action">
+              <button class="btn btn-primary-outlined">move to wishlist</button>
+              <button class="btn">remove from cart</button>
```

Comment:
give margin between the 2 buttons. also you can adds effects if button is hovered or clicked to enhance site experience